torch_xla/csrc/torch_util.h

```diff
@@ -59,6 +60,10 @@ at::Scalar MakeFloatScalar(T value) {
 // Unwraps tensor to target dtype if it's a wrapped number.
 at::Tensor UnwrapNumber(const at::Tensor& tensor, at::ScalarType dtype);
 
+// Wraps tensor to functional tensor if XLA_DISABLE_FUNCTIONALIZATION is false
+// or not set.
+at::Tensor MaybeWrapTensorToFunctional(const at::Tensor& tensor);
```

Comment:
Can you add a comment:
```
For unwrapping, `torch::lazy::maybe_unwrap_functional()` will only unwrap tensors that are functional. So, nothing needs to be done there.
```

Reply:
Added, thanks for the suggestion -- very helpful comment.